Doc/library/os.rst

```diff
@@ -1180,8 +1180,17 @@ as internal buffering of data.
           SEEK_CUR
           SEEK_END
 
-   Parameters to the :func:`lseek` function. Their values are 0, 1, and 2,
-   respectively.
+   Parameters to the :func:`lseek` function and the :meth:`~io.IOBase.seek`
+   method on file object, for whence to adjust the file position indicator.
```

Comment:
Should we only suggest these constants for functions in ``os``? (Note ``lseek`` talks about files, but ``io`` is more general in terms of streams)

```suggestion
   Parameters to the :func:`lseek` function,
   for whence to adjust the file position indicator.
```

Otherwise you need a plural objects:

```suggestion
   Parameters to the :func:`lseek` function and the :meth:`~io.IOBase.seek`
   method on file objects, for whence to adjust the file position indicator.
```

Reply:
I'm not sure; quoting one of the first sentences in the `io` docs:

> A concrete object belonging to any of these categories is called a [file object](https://docs.python.org/3/glossary.html#term-file-object). Other common terms are stream and file-like object.

Reply:
```suggestion
   Parameters to the :func:`lseek` function and the :meth:`~io.IOBase.seek`
   method on streams and :term:`file-like objects <file object>`, for whence to adjust the file position indicator.
```

Reply:
That was the poorer side of the rationale sorry, I should've been more focussed. The basic position is that the same constants exist in ``io`` and ``os``, and the question is do we want to encourage users of ``f.seek`` to use the ``io`` constants or the ``os`` constants.

My suggestion would be that yes, we should bifurcate, but I would understand wanting to mention the cross-compatability.

A

Reply:
Your suggestion wrapped:
```suggestion
   Parameters to the :func:`lseek` function and the :meth:`~io.IOBase.seek`
   method on streams and :term:`file-like objects <file object>`,
   for whence to adjust the file position indicator.
```

Reply:
*bifurcate*! TIL 😄

I think we should promote the `os` constants.

Reply:
> bifurcate

When such fun words exist, why not use them!

> I think we should promote the ``os`` constants.

In which case I agree with your suggestion!